codebase/analysis/src.lib/igrf_v2/igrf.1.0/src/igrflib.c

```diff
@@ -1250,23 +1256,18 @@ int geod2geoc(double lat, double lon, double alt, double rtp[]) {
 
 int plh2xyz(double lat, double lon, double alt, double rtp[])
 {
-  double a,b,f,ee,st,ct,sp,cp,N,Nac,x,y,z,r,t;
+  double a,f,ee,st,ct,N,Nac,z,r,t;
 
   a = 6378.1370;              /* semi-major axis */
   f = 1./298.257223563;       /* flattening */
-  b = a*(1. -f);              /* semi-minor axis */
```

Comment:
Should run this by @sshepherd before deleting his code (goes for other aacgm_v2 files as well).

Reply:
Believe this note is outdated now with 063ed65, right @egthomas?